packages/botonic-cli/src/util/bot-config.ts

```diff
@@ -12,42 +12,46 @@ const NPM_DEPTH_0 = 0
 // Also get alpha and beta versions
 const versionRegex = /@botonic\/[^@]+@(\d+\.\d+\.\d+(-[a-zA-Z]+\.\d+)?)/
 
-export interface BotConfigJson {
+type BotonicDependencies = Record<string, { version: string }>
+
+export interface BotConfigJSON {
   build_info: {
     node_version: string
     npm_version: string
     botonic_cli_version: string
   }
-  packages: Record<string, any>
+  packages: BotonicDependencies
 }
 export class BotConfig {
-  constructor(public appDirectory: string) {}
-
-  async createJson(): Promise<BotConfigJson> {
-    const packages = await this.readPackageJson()
+  static async get(appDirectory: string): Promise<BotConfigJSON> {
+    const spinner = ora({
+      text: 'Getting bot config...',
```

Comment:
i would remove this message, I think it is not needed for the user to know this is happening in the background